<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>number</class>
 <widget class="QDialog" name="number">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Missing Digit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/problem-solving.png</normaloff>:/problem-solving.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: black;
    color: white;
    border-radius: 10px;
    text-align: center;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEditNum">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>275</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="number_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Enter Number: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxType">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>275</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>ISBN - 10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ISBN - 13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Local Visa/MasterCard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>International Visa/MasterCard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Universal Product Code</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="missingDigit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Find Missing Digit</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
